docs/best-practices/repository.md

```diff
@@ -2,9 +2,18 @@
 
 - Use [this template](https://github.com/equinor/terraform-module-template) when creating your repository.
 
-- Use the common naming convention `terraform-azurerm-<name>` when naming your repository.
+- Use the common naming convention `terraform-azurerm-<module>` when naming your repository, where `<module>` is the name of the module.
 
-    For example, if you want to create a module named `storage`, the repository should be named `terraform-azurerm-storage`.
+    Modules should be named after the corresponding Azure CLI group or subgroup, for example:
+
+    - Terraform module [`key-vault`](https://registry.terraform.io/modules/equinor/key-vault/azurerm/latest) corresponds to Azure CLI group [`keyvault`](https://learn.microsoft.com/en-us/cli/azure/keyvault?view=azure-cli-latest).
+
+    - Terraform module [`storage`](https://registry.terraform.io/modules/equinor/storage/azurerm/latest) corresponds to Azure CLI group [`storage`](https://learn.microsoft.com/en-us/cli/azure/storage?view=azure-cli-latest).
+
+    - Terraform module [`log-analytics`](https://registry.terraform.io/modules/equinor/log-analytics/azurerm/latest) corresponds to Azure CLI subgroup [`log-analytics`](https://learn.microsoft.com/en-us/cli/azure/monitor/log-analytics?view=azure-cli-latest).
+
+!!! note
+    Azure CLI uses inconsistent separation of words in group names. We choose to consistenly separate words by `-` in module names.
```

Comment:
Do we have a naming convention for modules when we need more than one module, of the same type?

For example, we have one `storage` module and we need a second storage module for e.g. SQL. Would the naming convention then be `storage_sql` for the second storage module, or would the naming be something else?

This scenario can also be applied to App Service Plan, Function App, etc., to just give some examples on the scope of the use case for a naming convention here.

Reply:
Adding to that: the Terraform Baseline defines how we implement modules, not how we use them 🙂

Reply:
I agree with the way you put it here. Then its a thumbs up from me as well 👍🏼